runtime/kubernetes/image.go

```diff
@@ -29,8 +30,21 @@ const imagePatch = `
 
 // CreateImage creates the pipeline container image.
 func (c *client) CreateImage(ctx context.Context, ctn *pipeline.Container) error {
-	c.Logger.Tracef("no-op: creating image for container %s", ctn.ID)
+	c.Logger.Tracef("creating image for container %s", ctn.ID)
 
+	// parse/validate image from container
+	//
+	// https://pkg.go.dev/github.com/go-vela/worker/internal/image#ParseWithError
+	_, err := image.ParseWithError(ctn.Image)
+	if err != nil {
+		return err
+	}
+
+	// Kubernetes does not have an API to make sure it can access an image,
+	// so we have to query the appropriate docker registry ourselves.
+	// TODO: query docker registry for the image (if possible)
```

Comment:
Does it make sense to pull this TODO out into a story?

Reply:
Yeah. That'll be involved because we need whatever credentials are used with the (private in my case) docker registry. So either pull it from k8s or the admin has to configure Vela with it.

Do you mean put it in an issue on the community repo?

Reply:
By story, do you mean an issue on the community repo?

Reply:
Yeah, sorry poor phrasing

Reply:
Done: https://github.com/go-vela/community/issues/579